examples/uptime_sla/test/UptimeSLA_test.js

```diff
@@ -107,9 +106,11 @@ contract('UptimeSLA', () => {
       });
 
       it("does not accept the data provided", async () => {
-        await assertActionThrows(async () => {
-          await oc.fulfillData(requestId, response, {from: oracleNode})
-        });
+        let originalBalance = await eth.getBalance(sla.address)
+        await oc.fulfillData(requestId, response, {from: oracleNode})
+        let newBalance = await eth.getBalance(sla.address)
```

Comment:
The given value for `response` here is "1018956", and the time hasn't passed at this point, so the balance wouldn't change even if it did accept the fulfillment data in this test. It might be better to test that the value for `_uptime` is not equal to `uptime` after calling `oc.fulfillData`.

Reply:
Great point.